template/python3-http/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM python:3.7-alpine
```

Comment:
Why base python3-http on 3.7 and python-http-armhf on 3.6?
I think it would be a good idea to sync the two of them